components/AlarmTimer/index.js

```diff
@@ -0,0 +1,59 @@
+import { useEffect } from "react";
+
+import React from "react";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+export default function TimerComponent({ todaysEvents, onToggleAlarm }) {
+  const alarmEvents = todaysEvents.filter((event) => event.alarm === true);
+
+  function alarmVoiceMessage(eventNames) {
+    var msg = new SpeechSynthesisUtterance();
+    var voices = window.speechSynthesis.getVoices();
```

Comment:
this is a first for me 🗣️ I did not know that this API existed and now my computer is talking to me 😨